src/validations/rules/Writing_FedRAMP_Validations.md

```diff
@@ -0,0 +1,60 @@
+# How to create FedRAMP OSCAL validation constraints
+
+FedRAMP OSCAL documents are structured XML documents. The document syntax is defined by OSCAL XML Schema. However, XML Schema cannot express many semantic constraints, and an additional schema language is employed: Schematron.
+
+# Technologies used
+
+- [XML](https://www.w3.org/standards/xml/) is a structered document langauge.
+- [Namespaces in XML](https://www.w3.org/TR/xml-names/) are freqently encountered in structured XML documents
```

Comment:
Consider me a novice developer, why do I care?

Reply:
Namespaces routinely confuse novice developers and are a major cause of mistakes.

It's a bit late to excise XML namespace prefixes from what we've put together. I suppose I could just not mention them.

Reply:
Altered in https://github.com/18F/fedramp-automation/commit/bd71afaefcda11214aac1b0a6fd84d7f3e9a7762